plugins/tracers/gstctf.c

```diff
@@ -551,8 +551,13 @@ create_ctf_path (gchar * dir_name)
 static void
 ctf_file_init (void)
 {
+  const gchar *env_file_buf_value;
+  gchar *env_file_buf_value_end;
   gchar *metadata_file = NULL;
   gchar *datastream_file = NULL;
+  gint file_buf_size;
```

Comment:
[setvbuf](http://www.cplusplus.com/reference/cstdio/setvbuf/) receives a `size_t` (`g_size`). Let's use that.